web/themes/custom/usagov/usagov.libraries.yml

```diff
@@ -8,6 +8,7 @@ global-styling:
     scripts/uswds-init.min.js: {}
     scripts/newFooter.js: {}
     scripts/phoneForMobile.js: {}
+    scripts/callCenterWaitTimeLookup.js: {}
```

Comment:
Let's comment this out for now.

Reply:
Moving this to USAGOV-1945 with the other call center changes.